README.md

```diff
@@ -5,495 +5,21 @@
 
 ### If you are looking for the v18.2 branch, please follow this link: [https://github.com/DevExpress/devextreme-vue/tree/18.2](https://github.com/DevExpress/devextreme-vue/tree/18.2).
 
```

Comment:
Let's add a beautiful picture here. Sergey Z. have agreed that it's a good idea.

Reply:
Let's do this in another PR.